### ISSUE TITLE
DevOps - auto-assignment for accessibility tickets

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#9375]
Any ticket with the "accessibility" label, at the point that label is added, should be auto-assigned to `brea11y` as our accessibility specialist.

She requested this in a QART meeting, so she can be aware of accessibility-related work/issues as they move through.
